src/provider/states/stopping.rs

```diff
@@ -15,10 +15,9 @@ pub struct Stopping;
 impl State<PodState> for Stopping {
     async fn next(self: Box<Self>, pod_state: &mut PodState, _pod: &Pod) -> Transition<PodState> {
         if let Some(child) = &pod_state.process_handle {
```

Comment:
child is unused if you delete the pid line below but do you want to delete that line?